src/app/medical-record/medical-record.component.ts

```diff
@@ -253,7 +253,7 @@ export class MedicalRecordComponent implements OnInit {
   }
 
   downloadPDF( configName: string ) {
-    if (configName === 'cover' && this.medicalRecord.name == null || this.medicalRecord.name === "") {
+    if (configName === "cover" && this.medicalRecord.name == null || this.medicalRecord.name === "") {
```

Comment:
`this.medicalRecord.name == null || this.medicalRecord.name === ""` can be shortened to `!this.medicalRecord.name`